threediresultstyler/README.txt

```diff
@@ -0,0 +1,32 @@
+Plugin Builder Results
```

Comment:
Kan je dit niet beter leeglaten (of, beter,  vervangen door tekst over je plugin)?